src/FastRoute/include/fastroute/GlobalRouter.h

```diff
@@ -51,9 +51,22 @@ namespace ord {
 class OpenRoad;
 }
 
+namespace odb {
+class dbDatabase;
+class dbChip;
```

Comment:
you really don't have any reason to need a pointer to an dbChip so remove the class def

Reply:
dbChip removed